_posts/2020-08-20-flink-docker.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "The State of Flink on Docker"
+date: 2020-08-08T00:00:00.000Z
+authors:
+- rmetzger:
+  name: "Robert Metzger"
+  twitter: rmetzger_
+categories: news
+
+excerpt: This blog post gives an update on the recent developments of Flink's support for Docker.
+---
+
+The Flink community recently put some effort into upgrading the Docker experience for our users. The goal was to reduce confusion and improve usability. With over 50 million downloads from Docker Hub, the Flink docker images are a very popular deployment option.
+
+Let's quickly break down the recent improvements:
+
+- Reduce confusion: Flink used to have 2 Dockerfiles and a 3rd file maintained outside of the official repository — all with different features and varying stability. Now, we have one central place for all images: [apache/flink-docker](https://github.com/apache/flink-docker).
+
+  Here, we keep all the Dockerfiles for the different releases. Check out the [detailed readme](https://github.com/apache/flink-docker/blob/master/README.md) of that repository for further explanation on the different branches, as well as the [Flink Improvement Proposal (FLIP-111)](https://cwiki.apache.org/confluence/display/FLINK/FLIP-111%3A+Docker+image+unification) that contains the detailed planning.
+
+  The apache/flink-docker repository also seeds the o[fficial Flink image on Docker Hub](https://hub.docker.com/_/flink).
+
+- Improve Usability: The Dockerfiles are used for various purposes: [Native docker deployments](https://ci.apache.org/projects/flink/flink-docs-master/ops/deployment/docker.html), [Flink on Kubernetes](https://ci.apache.org/projects/flink/flink-docs-master/ops/deployment/native_kubernetes.html), the (unofficial) [Flink helm example](https://github.com/docker-flink/examples) and the project's [internal end to end tests](https://github.com/apache/flink/tree/master/flink-end-to-end-tests). With one unified image, all these consumers of the images benefit from the same set of features, documentation and testing. 
+
+  The new images support passing configuration variables via a `FLINK_PROPERTIES` environment variable. Users can enable default plugins with the `ENABLE_BUILT_IN_PLUGINS`environment variable. The images also allow loading custom jar paths and configuration files.
```

Comment:
It may be helpful to turn the phrases for key features into links to the corresponding doc chapters (e.g. FLINK_PROPERTIES, ENABLE_BUILT_IN_PLUGINS, custom jar paths and configuration files). I know we already have a general link but I believe more links do not hurt.